src/Resources/views/Block/block_stats.html.twig

```diff
@@ -17,10 +17,10 @@ file that was distributed with this source code.
     <!-- small box -->
     <div class="small-box {{ settings.color }}">
         <div class="inner">
-            <h3>{{ pager.count() }}</h3>
+            <h3>{{ pager.getResults() }}</h3>
```

Comment:
Shouldn't this be `pager.getNbResults()`?

Reply:
BTW, "count" seems a proper name IMO. Even, it is consistent with the [`Countable`](https://www.php.net/manual/es/countable.count.php) interface.

Reply:
> Shouldn't this be `pager.getNbResults()`?

Yes, indeed.

> BTW, "count" seems a proper name IMO. Even, it is consistent with the [`Countable`](https://www.php.net/manual/es/countable.count.php) interface.

I want to remove the `Countable` interface from the `AbstractPager`, and the method `count`.
There is no need for having two methods `getNbResults` and `count` doing the same thing.

Since the `PagerInterface` is asking for a `getNbResults()` method (which has a clearer name than `count` ; when I read `pager->count()`, it's not clear to me that we are counting the results, we could have counted the pages for instance.) and is not implementing Countable, our template should only use `getNbResults` in order to work with any PagerInterface, and not only those who extends the AbstractPager.
In order to avoid this king of error, we should remove from the AbstractPager all the methods which are not in the Interface. Moreover, most of them are not used.

Reply:
I'm sorry to disagree. "count" seems clearer than "getNbResults" to me. I understand it may be not implementing `\Countable`, I just mentioned that for consistency reasons about the method name, given its purpose.
If "count" is not clear for you, maybe we could evaluate "countResults" or something like that.

Reply:
> I'm sorry to disagree. "count" seems clearer than "getNbResults" to me.

A Pager have pages with results. It exposes multiples method `isFirstPage`, `getNextPage`, `getPreviousPage`, ... about page and only one `getNbResults` about results. So I see the pager like a book, where a result is a line. We're usually counting pages on a book.
So how is it clear to you here it'll return `getNbResults` and not something like `getNbPages` ?

And I think `getNbResult` is the method used by others bundle https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/21d223550299707a598a4e1433011c3a8cfb6cff/src/Datagrid/Pager.php#L87

> I understand it may be not implementing `\Countable`, I just mentioned that for consistency reasons about the method name, given its purpose.
> If "count" is not clear for you, maybe we could evaluate "countResults" or something like that.

Things is that `getNbResults` already is in the interface, and is used by other bundles.
You can basically see this change (count into getNbResults) as a bugfix. If it was PHP code, it would be reported by Phpstan/Psalm.

Introducing `count`/`countResults` instead of `getNbResults` add supplementary work, for this bundle and others.

Reply:
Are you ok with my last message @phansys ?

The reason of the change from `count` to `getNbResults` is that the Pager doesn't implement \Countable.

Reply:
I understand it doesn't implement `\Countable`. I'm just saying that `count()` seems more accurate or standardized than `getNbResults()`.

Reply:
But what do you advice ?

Overall `pager.getNbResults` was always the methods used in SonataAdmin (and in other library like https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/21d223550299707a598a4e1433011c3a8cfb6cff/src/Datagrid/Pager.php#L87). 

Keeping `getNbResults` seems to me the way to avoid extra rework.

And I still find `count` not intuitive. Why would it count the results and not the pages ?
Then you proposed `countNbResults`, but it would lead to extra work to deprecate `getNbResults`.
I'd like to minimize the work to do before 4.0.

I understand you point of view but I clearly don't know what to do here.

Reply:
>Then you proposed countNbResults, but it would lead to extra work to deprecate getNbResults.

I proposed `countResults()`, not `countNbResults()`, since the "nb" term / concept seems extremely opinionated to me.
I was expecting to avoid an extra PR with that renaming, but I don't want to block this one either. If you think there is no choice to apply that change in this PR, I'll try to create a new one later.

Reply:
@phansys Is it ok for you if I create an issue to remind me/us to rename the method ?

This is already a big PR. I prefer to do the renaming in another.

Reply:
Sure @VincentLanglet, thank you.

Reply:
Done https://github.com/sonata-project/SonataAdminBundle/issues/6710

If this PR is ok for you, I would prefer your approval (to be sure). :)